fetch-in-react/src/app.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import UserList from './user-list';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      users: [],
+      isLoading: true
+    };
+  }
+
+  componentDidMount() {
+    /* your code here */
+    this.setState({ isLoading: false });
```

Comment:
It's better to wait until your fetch is actually completed to update this state.